aws/requirements.txt

```diff
@@ -1,12 +1,12 @@
-boto3==1.21.10
-botocore==1.24.10
-certifi==2021.10.8
-charset-normalizer==2.0.12
-docutils==0.18.1
-idna==3.3
-jmespath==0.10.0
+boto3==1.25.5
+botocore==1.28.5
+certifi==2022.9.24
+charset-normalizer==2.1.1
+idna==3.4
+jmespath==1.0.1
 python-dateutil==2.8.2
-requests==2.27.1
-s3transfer==0.5.2
+requests==2.28.1
+s3transfer==0.6.0
 six==1.16.0
-urllib3==1.26.8
+urllib3==1.26.12
+PyYAML==6.0
```

Comment:
The `PyYAML` requirement is only used for the `cleanup.py` script, so shouldn't be included here. Including it here will put it into the venv which gets packaged up for the Lambda function, which isn't needed.

Perhaps the requirement should go into a separate dev-requirements.txt file.